README.md

```diff
@@ -370,12 +370,14 @@ client
 
 ##### Create a Signed Search Key
 
-Creating a search key that will only return the title field.
+Signed search keys keep your read-only Private API Keys secret and restrict what a user can search over. They can be used to restrict top-level search options. The example below creates a signed search key that will only return the `title` field.
```

Comment:
```suggestion
Signed search keys keep your read-only private API keys secret and restrict what a user can search over. They can be used to restrict top-level search options. The example below creates a signed search key that will only return the `title` field.
```

Just a minor suggestion to change [capitalization of feature names](https://brand.elastic.co/302f66895/p/072ccc-naming-guide). Else, everything LGTM 👍  Thanks for submitting this doc change!

Reply:
FYI that was copied from the linked Elastic docs, so we should update it there too